4-storage/relational-database-shredder/src/main/scala/com.snowplowanalytics.snowplow.storage/spark/inputs/EnrichedEventLoader.scala

```diff
@@ -1,49 +1,40 @@
 /*
- * Copyright (c) 2014 Snowplow Analytics Ltd. All rights reserved.
+ * copyright (c) 2012-2017 snowplow analytics ltd. all rights reserved.
  *
- * This program is licensed to you under the Apache License Version 2.0,
- * and you may not use this file except in compliance with the Apache License Version 2.0.
- * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ * this program is licensed to you under the apache license version 2.0,
+ * and you may not use this file except in compliance with the apache license version 2.0.
+ * you may obtain a copy of the apache license version 2.0 at
+ * http://www.apache.org/licenses/license-2.0.
  *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the Apache License Version 2.0 is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ * unless required by applicable law or agreed to in writing,
+ * software distributed under the apache license version 2.0 is distributed on an
+ * "as is" basis, without warranties or conditions of any kind, either express or implied.
+ * see the apache license version 2.0 for the specific language governing permissions and
+ * limitations there under.
  */
-package com.snowplowanalytics.snowplow.enrich
-package hadoop
-package inputs
+package com.snowplowanalytics.snowplow.storage.spark.inputs
 
-// Java
 import java.util.UUID
 
-// Joda-Time
+import com.snowplowanalytics.snowplow.enrich.common.{ValidatedEnrichedEvent, ValidatedString}
+import com.snowplowanalytics.snowplow.enrich.common.outputs.EnrichedEvent
+import com.snowplowanalytics.util.Tap._
```

Comment:
Can we use this opportunity to get rid of Tap? It was a bad idea by me to introduce it to Snowplow years ago and it's easily replaceable with an idiomatic Scala block.